app_dart/lib/src/request_handlers/file_flaky_issue_and_pr.dart

```diff
@@ -118,6 +112,24 @@ class FileFlakyIssueAndPR extends ApiRequestHandler<Body> {
     await gitHub.assignReviewer(slug, reviewer: prBuilder.pullRequestReviewer, pullRequestNumber: pullRequest.number);
   }
 
+  bool _shouldNotFileIssueAndPR(_BuilderDetail builderDetail, Issue? issue) {
+    // Don't create a new issue or deflake PR if the builder has been marked as flaky.
```

Comment:
This comment seems inaccurate. Maybe mentions this is handled in other request_handler.

Reply:
Good catch! Updated.